### ISSUE TITLE
Output in TAP format

### ISSUE THREAD

**Issue by kulseran** [ollelogdahl/ihct#1]
https://en.wikipedia.org/wiki/Test_Anything_Protocol

TAP is a machine readable output format which allows many other tools (CI frameworks) to properly parse the output of the test run.

**Comment by ollelogdahl** [comment]
Yes this could be possible. I'm thinking of adding TAP support with a argument flag on calling the test executable (`-t` for example). But the thing is that a test unit is terminated on the first failed assertion. Also, there is no way to assign a string of text for each assert. This framework doesn't really have the required developer interface for this. As said, it will be considered.
